ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -279,7 +279,6 @@ def write_topics(self, topic: Topic) -> None:
         try:
             topic_request = CreateTopicRequest(
                 name=topic.name,
-                displayName=topic.displayName,
```

Comment:
Checked the JSON Schema on 0.10.4 and it does not have displayName. Might be from another patch